testsuite/integration-arquillian/servers/adapter-spi/undertow-adapter-jakarta/pom.xml

```diff
@@ -83,12 +83,10 @@
         <dependency>
             <groupId>io.undertow</groupId>
             <artifactId>undertow-servlet</artifactId>
-            <version>${undertow-jakarta.version}</version>
         </dependency>
         <dependency>
             <groupId>io.undertow</groupId>
             <artifactId>undertow-core</artifactId>
-            <version>${undertow-jakarta.version}</version>
         </dependency>
         <dependency>
             <groupId>junit</groupId>
```

Comment:
Just out of curiosity, why do we need the `-jakarta` adapter packages? No adapters should use Jakarta EE, AFAIK.